### PR TITLE
compass-ui serve mainframe and content app

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Expose multiple react apps (mainframe and content) on 2 ports 

**Related issue(s)**
https://github.com/kyma-incubator/compass/issues/152

### CODE REVIEW

File: compass/src/config/config.js [hunk 1]
```diff
@@ -1 +1,6 @@
-export const COMPASS_GRAPHQL_ENDPOINT = `https://compass-gateway.kyma.local/director/graphql`;
+window.clusterConfig = {
```
Comment (sjanota) [inline compass/src/config/config.js]: Why there are two `config.js`es? another one is in `public/`

File: compass/public/config/config.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+window.clusterConfig = {
+    microfrontendContentUrl : 'http://localhost:8888',
+    graphqlApiUrl: 'https://compass-gateway.kyma.local/director/graphql',
+    tenants: 'A B C'
```
Comment (sjanota) [inline compass/public/config/config.js]: There is no `defaultTenant`

File: compass/src/commons/builder.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import LuigiClient from '@kyma-project/luigi-client';
 const uuidv5 = require('uuid/v5');
+var validate = require('uuid-validate');
```
Comment (dariadomagala) [inline compass/src/commons/builder.js]:
```suggestion
const validate = require('uuid-validate');
```

File: compass/src/commons/builder.js [hunk 4]
```diff
@@ -18,6 +19,9 @@ class Builder {
   }
 
   getTenant() {
+    if (validate(this.currentTenant)) {
```
Comment (dariadomagala) [inline compass/src/commons/builder.js]: Maybe just
```
getTenant() {
    return validate(this.currentTenant) ? this.currentTenant : uuidv5(this.currentTenant, NAMESPACE);
}
```
?

**Pull request merged**